### PR TITLE
[new release] Alt-Ergo 2.4.3

### PR DESCRIPTION
**Pull request by Halbaroth**
This pull-request concerns:
- alt-ergo.2.4.3: The Alt-Ergo SMT prover
- alt-ergo-lib.2.4.3: The Alt-Ergo SMT prover library
- alt-ergo-parsers.2.4.3: The Alt-Ergo SMT prover parser library
- altgr-ergo.2.4.3: The GUI for the Alt-Ergo SMT prover

Homepage: https://alt-ergo.ocamlpro.com/
Source repo: git+https://github.com/OCamlPro/alt-ergo.git
Bug tracker: https://github.com/OCamlPro/alt-ergo/issues

### CODE REVIEW

File: packages/alt-ergo-lib/alt-ergo-lib.2.4.3/opam [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+version: "2.4.3"
```
Comment (kit-ty-kate) [inline packages/alt-ergo-lib/alt-ergo-lib.2.4.3/opam:3]:
```suggestion
```

File: packages/alt-ergo-parsers/alt-ergo-parsers.2.4.3/opam [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+version: "2.4.3"
```
Comment (kit-ty-kate) [inline packages/alt-ergo-parsers/alt-ergo-parsers.2.4.3/opam:3]:
```suggestion
```

**Comment by Halbaroth** [comment]
It's ready :)

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**